policyengine_us_data/datasets/cps/cps.py

```diff
@@ -730,6 +731,23 @@ def add_personal_income_variables(
     cps["other_medical_expenses"] = person.PMED_VAL
     cps["medicare_part_b_premiums"] = person.PEMCPREM
 
+    # ------------------------------------------------------------------
+    # Qualified Business Income deduction flags. These indicate whether
+    # the corresponding stream of income would qualify for the 199A
+    # deduction if it were present.
+    QBI_QUALIFICATION_PROBABILITIES = {
```

Comment:
Source?

Reply:
@MaxGhenis , do you have the O3 chat link handy for this guy?
![image](https://github.com/user-attachments/assets/1c0e4cff-058f-4a6b-95f5-12ca126354a9)

Reply:
Ah, it's a discussion, and the O3 chat is linked! https://github.com/PolicyEngine/policyengine-us-data/discussions/237

Adding it now..

Reply:
I added the chat source and the discussion number. (What's showing in the preview is outdated)

Reply:
The raw data is now in utils/soi.py by the way (since it's now used in 2 places)